src/form/Calendar/CalendarDays/CalendarDays.tsx

```diff
@@ -89,15 +93,22 @@ export interface CalendarDaysProps {
   range?: [Date, Date] | [Date, undefined] | [undefined, undefined];
 
   /**
+   * @deprecated Use animation configuration instead.
    * X-axis block animation
    */
   xAnimation?: string | number;
 
   /**
+   * @deprecated Use animation configuration instead.
    * Whether to animate the calendar.
    */
   animated?: boolean;
 
+  /**
+   * Animation configuration for the calendar days.
+   */
+  animation?: MotionNodeAnimationOptions;
```

Comment:
I don't love the name `animation` because its more like a `transition` but `transition` is already a thing in framer so prob should not use that either.

Reply:
No I dont think so, I dont want to tie it to the library or 'implementation' in this case.